iml-request-retry/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "iml-request-retry"
+version = "0.1.0"
+description = "IML async request retry Library"
+authors = ["IML Team <iml@whamcloud.com>"]
+edition = "2018"
+
+[dependencies]
+futures-preview = "0.3.0-alpha.19"
+tokio = "0.2.0-alpha.6"
+rand = "0.7.2"
+tracing = "0.1.10"
+
+[dev-dependencies]
+tokio-test = "0.2.0-alpha.6"
+http = "0.1.19"
+reqwest = { version = "0.10.0-alpha.1", features = ["json"] }
+url = "2.1.0"
+rand_xorshift = "0.2.0"
+warp = { version = "0.2.0-alpha.0", git = "https://github.com/seanmonstar/warp.git", rev = "9bb58102"}
+serde = "1.0.102"
```

Comment:
This can be:

`serde = { version = "1", features = ['derive'] }`

and line 23 can be dropped.

Reply:
This fix in `Cargo.toml` caused the failure on the CI, that I cannot reproduce locally. Just weird 🔨. However, I have found (in the reddit /r/rust), that the issue is fixed by
```
- use serde_derive::{Serialize, Deserialize};
+ use serde::{Serialize, Deserialize};
```
This might be reproduced, if one builds it in a docker container. Ugh :-(